src/Microsoft.ML.Data/Transforms/ColumnCopying.cs

```diff
@@ -31,8 +31,22 @@
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// <see cref="ColumnCopyingEstimator"/> copies the input column to another column named as specified in the parameters of the transformation.
+    /// Estimator creating the <see cref="ColumnCopyingTransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ### Estimator FAQ
+    /// | Question | Answer |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type. | Any |
```

Comment:
>. [](start = 32, length = 1)

is the period necessary?

Reply:
if the above have a ? feels natural to have a . here. IDK, @natke?

---
In reply to: [275070222](https://github.com/dotnet/machinelearning/pull/3316#discussion_r275070222) [](ancestors = 275070222)

Reply:
I'd say no to the "." :-)